File: app/src/main/java/net/devdome/paperplayer/data/model/SongList.java
package net.devdome.paperplayer.data.model;

import java.util.ArrayList;
import java.util.List;

/**
 * PaperPlayer
 * Michael Obi
 * 22 10 2016 2:57 AM
 */

public class SongList {

    private List<Song> songs;

    public SongList(List<Song> songs) {
        this.songs = songs;
    }

    public List<Song> getSongs() {
        return songs;
    }

    public void setSongs(ArrayList<Song> songs) {
        this.songs = songs;
    }
}

File: app/src/main/java/xyz/michaelobi/paperplayer/data/model/SongList.kt
package xyz.michaelobi.paperplayer.data.model

import java.util.*

/**
 * PaperPlayer
 * Michael Obi
 * 22 10 2016 2:57 AM
 */

class SongList(songs: List<Song>) {

    var songs: List<Song>? = null
        private set

    init {
        this.songs = songs
    }

    fun setSongs(songs: ArrayList<Song>) {
        this.songs = songs
    }
}
